Source/WebCore/page/SecurityOriginData.cpp

```diff
@@ -66,9 +67,23 @@ SecurityOriginData SecurityOriginData::fromFrame(Frame* frame)
     return document->securityOrigin().data();
 }
 
+SecurityOriginData SecurityOriginData::fromURL(const URL& url, ShouldSkipOpaqueSecurityOriginCheck shouldSkipOpaqueSecurityOriginCheck)
+{
+    bool shouldCreateOpaqueOrigin = ShouldSkipOpaqueSecurityOriginCheck::No == shouldSkipOpaqueSecurityOriginCheck && shouldTreatAsOpaqueOrigin(url);
```

Comment:
I personally think that `shouldSkipOpaqueSecurityOriginCheck  == ShouldSkipOpaqueSecurityOriginCheck::No` looks better than `ShouldSkipOpaqueSecurityOriginCheck::No == shouldSkipOpaqueSecurityOriginCheck`.